staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -171,10 +190,34 @@ func (t *prefixTransformers) TransformToStorage(ctx context.Context, data []byte
 	result, err := transformer.Transformer.TransformToStorage(ctx, data, dataCtx)
 	RecordTransformation("to_storage", string(transformer.Prefix), time.Since(start), err)
 	if err != nil {
+		if klogLevel6 := klog.V(6); klogLevel6.Enabled() {
```

Comment:
Can you make a `logTransformErr(message)` function that uses `VDepth` to get the correct line and avoids this duplicated code?

Reply:
Oh I missed https://github.com/kubernetes/kubernetes/pull/119166#discussion_r1267139878 - now I recall this is why we have the duplicated code in the other place.  I am fine with the duplication to avoid fighting the linter.

Reply:
Actually, I made linters happy while creating a common function.